packages/pocket-ponics-app/screens/tier-screen.js

```diff
@@ -75,6 +87,14 @@ export default class Example extends React.Component {
 		return "Adjusting"
 	}
 
+	harvestinstruction() {
+		const resetAction = StackActions.reset({
+			index: 0,
+			actions: [NavigationActions.navigate({ routeName: 'HarvestInstruction' })],
+		});
+		this.props.navigation.dispatch(resetAction);
+	}
```

Comment:
We can probably use the props.navigation.navigate option here, since harvesting isn't a multi-screen process that needs to block the Back button.

Reply:
Hey, thanks so much for the feedback! I just wanted to let you know that the back button is not blocked, so I'm going to go ahead and leave this one the way it is. I am working on making all the other necessary changes that you have requested!